packages/jaeger-ui/src/components/TracePage/TraceStatistics/index.tsx

```diff
@@ -392,79 +263,168 @@ export default class TraceStatistics extends Component<Props, State> {
+      },
+      {
+        title: 'Avg',
+        dataIndex: 'avg',
+        sorter: (a, b) => sorterFunction({"type": a.type,"index": a.avg},{"type": b.type,"index": b.avg}),
+        render: (cell: string) => {
+              return cell+'ms'
+        },
+        onCell: (record) => onCellFunction(record)
+      },
+      {
+        title: 'Min',
+        dataIndex: 'min',
+        sorter: (a, b) => sorterFunction({"type": a.type,"index": a.min},{"type": b.type,"index": b.min}),
+        render: (cell: string) => {
+              return cell+'ms'
+        },
+        onCell: (record) => onCellFunction(record)
+      },
+      {
+        title: 'Max',
+        dataIndex: 'max',
+        sorter: (a, b) => sorterFunction({"type": a.type,"index": a.max},{"type": b.type,"index": b.max}),
+        render: (cell: string) => {
+              return cell+'ms'
+        },
+        onCell: (record) => onCellFunction(record)
+      },
+      {
+        title: 'ST Total',
+        dataIndex: 'selfTotal',
+        sorter: (a, b) => sorterFunction({"type": a.type,"index": a.selfTotal},{"type": b.type,"index": b.selfTotal}),
+        render: (cell: string) => {
+              return cell+'ms'
+        },
+        onCell: (record) => onCellFunction(record)
+      },
+      {
+        title: 'ST Avg',
+        dataIndex: 'selfAvg',
+        sorter: (a, b) => sorterFunction({"type": a.type,"index": a.selfAvg},{"type": b.type,"index": b.selfAvg}),
+        render: (cell: string) => {
+              return cell+'ms'
+        },
+        onCell: (record) => onCellFunction(record)
+      },
+      {
+        title: 'ST Min',
+        dataIndex: 'selfMin',
+        sorter: (a, b) => sorterFunction({"type": a.type,"index": a.selfMin},{"type": b.type,"index": b.selfMin}),
+        render: (cell: string) => {
+              return cell+'ms'
+        },
+        onCell: (record) => onCellFunction(record)
+      },
+      {
+        title: 'ST Max',
+        dataIndex: 'selfMax',
+        sorter: (a, b) => sorterFunction({"type": a.type,"index": a.selfMax},{"type": b.type,"index": b.selfMax}),
+        render: (cell: string) => {
+              return cell+'ms'
+        },
+        onCell: (record) => onCellFunction(record)
+      },
+      {
+        title: 'ST in Duration',
+        dataIndex: 'percent',
+        sorter: (a, b) => sorterFunction({"type": a.type,"index": a.percent},{"type": b.type,"index": b.percent}),
+        render: (cell: string) => {
+              return cell+'%'
+        },
+        onCell: (record) => onCellFunction(record)
+      },
+    ];
+    let isDetailArray: ITableSpan[] = [];
+    let isNoDetail: ITableSpan[] = [];
+  
+    const findTablesAccToSelectors = (tableValue: ITableSpan[]) => {
+      for (let i = 0; i < tableValue.length; i++) {
+        if (tableValue[i].isDetail) {
+          isDetailArray.push(tableValue[i]);
+        } else {
+          isNoDetail.push(tableValue[i]);
+        }
+      } 
```

Comment:
Let's reduce this to a simplified version - 
```suggestion
      isDetailArray = tableValue.filter((val : ITableSpan) => val.isDetail) 
      isNoDetail = tableValue.filter((val : ITableSpan) => !val.isDetail) 
```